stan-dev/stan
# Feature/issue 786 manual 2.5

#### Summary:

Update doc for 2.5
#### Intended Effect:

See above.  Adds chapter on algorithms and configuration, chapter on marginalizing discrete latent parameters,  updates the mathematical definitions, corrects all user-reported and dev-reported errors, and relative to last time, also adds ODE solver sections from other pull requests.
#### How to Verify:

Nothing to verify in code.  It would certainly be possible to read the doc and see if it's all right.  Most of the description was copied from Michael's and Marcus's writeups in the cmd-stan manual.  
#### Side Effects:

None.
#### Documentation:

It's all doc.
#### Reviewer Suggestions:

Anyone.
